### DISCUSSION TITLE
Questions on draw.io desktop

### DISCUSSION THREAD

**Discussion by volkiswag** [jgraph/drawio-desktop#1710] [Q&A] [answered]
Thank you Mr Benson for your help on my questions on app.diagramms.net. As you suggested I adapted the questions and post it on this forum again. I'm a trainee so I hope I dont put "usless" questions on the desktop version here.

1.	Please describe the systems architecture and the basic components of your platform. 

2.	Does your solution utilize cloud native services (>= PaaS, Microservices, etc.)?

3.	Which transport protocols & encryption do you use?

4.	Which functionalities of your solution are accessible via API?

5.	Are all data assets within your solution accessible via API?

6.	Are there 3rd party partners involved / get access / consume data of us out of your solution (e.g. google analytics)?

7.	Is it possible for us to audit and export the workflow & event management logs?

8.	Does your solution provide the capability to monitor from an End-to-End perspective?

9.	Do you refer on ITIL v3/ 4 standard?

10.	Please describe your update process for minor and major updates.

**Comment by davidjgraph** [comment, marked as answer]
Please describe the systems architecture and the basic components of your platform.

Core JS editor wrapped in electron.

    Does your solution utilize cloud native services (>= PaaS, Microservices, etc.)?

There is no cloud component.

    Which transport protocols & encryption do you use?

There is no comms from the app.

    Which functionalities of your solution are accessible via API?

type --help on the command line

    Are all data assets within your solution accessible via API?

You store files on your local filesystem.

    Are there 3rd party partners involved / get access / consume data of us out of your solution (e.g. google analytics)?

No

    Is it possible for us to audit and export the workflow & event management logs?

No

    Does your solution provide the capability to monitor from an End-to-End perspective?

Specifically?

    Do you refer on ITIL v3/ 4 standard?

No

    Please describe your update process for minor and major updates.

Look at the commit history in the repo.
